<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyEmployee</class>
 <widget class="QDialog" name="ModifyEmployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>401</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	color:white;
	font: 10pt &quot;Open Sans&quot;;
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #0b2e6f, stop:1 #003c8f);
}
QMessageBox{

}

QMessageBox QLabel {
	background-color:transparent;
    color: white;
}


QTextEdit {
    border: 1px solid qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #FF4B2B, stop:1 #FF416C);
    border-radius: 5px;
    padding: 5px 15px;
    background-color: white;
    color: black;
    font-size: 12px;
}

QTextEdit:focus {
    border: 1px solid qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #32CD32, stop:1 #00FFFF);
}

QTextEdit:disabled {
    background-color: #F0F0F0;
    color: #A0A0A0;
}

QTextEdit:read-only {
    background-color: #F0F0F0;
    color: #A0A0A0;
}

QTextEdit::hover {
    border: 1px solid qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #BEBEBE, stop:1 #A4A4A4);
}

QTextEdit::selected {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #FF4B2B, stop:1 #FF416C);
    color: white;
}

QLineEdit {
    border: 1px solid qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #FF4B2B, stop:1 #FF416C);
    border-radius: 5px;
    padding: 5px 15px;
    background-color: white;
    color: black;
    font-size: 12px;
}

QLineEdit:focus {
    border: 1px solid qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #32CD32, stop:1 #00FFFF);
}

QLineEdit:disabled {
    background-color: #F0F0F0;
    color: #A0A0A0;
}

QLineEdit:read-only {
    background-color: #F0F0F0;
    color: #A0A0A0;
}

QLineEdit::hover {
    border: 1px solid qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #BEBEBE, stop:1 #A4A4A4);
}

QLineEdit::selected {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #FF4B2B, stop:1 #FF416C);
    color: white;
}
QCheckBox {
    spacing: 10px;
	
	font: 12pt &quot;Open Sans Light&quot;;
}

QCheckBox::indicator {
    width: 35px;
    height: 35px;
}

QCheckBox::indicator:unchecked {
    image: url(:assets/icons/unchecked.png);
}

QCheckBox::indicator:checked {
    image: url(:assets/icons/checked.png);
}

QDateEdit {
    border: none;
    background-color: rgb(255, 255, 255);
    color: black;
    font-size: 16px;
    font-weight: normal;
    padding: 5px;
    border-radius: 5px;
}

QDateEdit::drop-down {
    background-color: transparent;
    image: url(:/assets/icons/cil-arrow-bottom_black.png);
    width: 20px;
    height: 20px;
}

QDateEdit::down-arrow {
    image: url(:/assets/icons/cil-arrow-bottom_black.png);
}

QDateEdit::up-arrow {
    image: url(:/assets/icons/cil-arrow-top_black.png);
}

QDateEdit::down-button {
    image: url(:/assets/icons/cil-arrow-bottom_black.png);
}

QDateEdit::up-button {
    image: url(:/assets/icons/cil-arrow-top_black.png);
}

QDateEdit::calendarWidget {
    border: none;
    background-color: rgb(255, 255, 255);
    color: black;
    font-size: 16px;
    font-weight: normal;
    padding: 5px;
    border-radius: 5px;
}

QDateEdit::section {
    background-color: transparent;
    border: none;
}

QPushButton {
    border: 1px solid qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #FF4B2B, stop:1 #FF416C);
    border-radius: 10px;
    padding: 5px 15px;
    background: #f9f9f9;
    color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #FF4B2B, stop:1 #FF416C);
    font-size: 12px;
}

QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #FF416C, stop:1 #FF4B2B);
	color: white;
}

QPushButton:pressed {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #FF4B2B, stop:1 #FF416C);
    border: 2px solid #FF416C;
    padding: 4px 14px 6px 16px;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>360</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Modify</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: white;
background-color: transparent;
}
QGroupBox{
	color: white;
}</string>
   </property>
   <property name="title">
    <string>General Information</string>
   </property>
   <widget class="QDateEdit" name="New_DOB_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="New_CIN_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Employee CIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>CIN:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Employee Last Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="New_LastName_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Employee Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="New_Name_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Employee Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>E-Mail:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Employee DOB:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Employee Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="New_Email_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Employee E-Mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Employee Phone:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phone">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="placeholderText">
     <string>Employee Phone Number</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: white;
background-color: transparent;
}
QGroupBox{
	color: white;
}
</string>
   </property>
   <property name="title">
    <string>Work Information</string>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Remarks:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Department:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="New_Type_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    border: 1px solid qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #FF4B2B, stop:1 #FF416C);
    border-radius: 5px;
    padding: 5px 15px;
    min-width: 100px;
    background-color: white;
    color: black;
    font-size: 12px;
}

QComboBox:drop-down {
    width: 20px;
    height: 20px;
	margin-top: 4px;
    margin-right: 5px;
    background-image: url(&quot;:assets/icons/cil-arrow-bottom_black.png&quot;);
    background-repeat: no-repeat;
    background-position: center center;
    border: none;
}

QComboBox:focus {
    border: 1px solid qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #32CD32, stop:1 #00FFFF);
}

QComboBox:disabled {
    background-color: #F0F0F0;
    color: #A0A0A0;
}

QComboBox:editable {
    background-color: white;
    border: 1px solid qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #BEBEBE, stop:1 #A4A4A4);
}

QComboBox:editable:focus {
    border: 1px solid qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #32CD32, stop:1 #00FFFF);
}

QComboBox::item {
    padding: 5px;
}

QComboBox::item:selected {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #FF4B2B, stop:1 #FF416C);
    color: white;
}</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Update/Add Face ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Face_ID_Btn_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="assets.qrc">
      <normaloff>:/assets/icons/face-id.png</normaloff>:/assets/icons/face-id.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Update/Add Fingerprint</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Face_ID_Btn_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="assets.qrc">
      <normaloff>:/assets/icons/cil-fingerprint_black.png</normaloff>:/assets/icons/cil-fingerprint_black.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>User Mode</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #FF4B2B, stop:1 #FF416C);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="assets.qrc">
      <normaloff>:/assets/images/logo.png</normaloff>:/assets/images/logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>654</x>
      <y>1</y>
      <width>155</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 255, 255);
font: 81 10pt &quot;Open Sans ExtraBold&quot;;</string>
    </property>
    <property name="text">
     <string>Employee Modification</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>4</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/assets/icons/cil-ban.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To manually reset your password, please contact us</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
